src/libstd/sys/common/backtrace.rs

```diff
@@ -36,7 +36,7 @@ pub fn log_enabled() -> bool {
     }
 
     let val = match env::var_os("RUST_BACKTRACE") {
-        Some(..) => 2,
+        Some(x) => if x.eq("disabled") { 1 } else { 2 },
```

Comment:
Should I transform this `x.eq("disabled")`
into this? `x.to_lowercase().eq("disabled")`
So that cases like `DisAbled` or `DISABLED` can also mean the same thing?!